src/node/rpc/endpoint.h

```diff
@@ -70,7 +72,8 @@ namespace ccf
       execute_locally,
       require_client_signature,
       require_client_identity);
-    DECLARE_JSON_OPTIONAL_FIELDS(EndpointProperties, openapi);
+    DECLARE_JSON_OPTIONAL_FIELDS(
+      EndpointProperties, require_jwt_authentication, openapi);
```

Comment:
Why is this optional?

Reply:
So that existing app.json's don't break.

Reply:
OK, that's what I suspected. Is the plan to move this to "required" once the JS apps are updated?

Reply:
With #1904 this will be replaced by something else.

Reply:
This is a good illustration of why expanding all those fields in the app.json was not a good idea...